README.md

```diff
@@ -1 +1,151 @@
-# Decode_Morse_Code
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>Decode Morse Code</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+- [📖 Decode Morse Code ](#-decode-morse-code-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Decode Morse Code <a name="about-project"></a>
+
+**Decode Morse Code** is an app to decode morse code using Ruby.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Built methods using Ruby**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO
+## 🚀 Live Demo <a name="live-demo"></a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby
+    bundler
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/K0ppai/Decode_Morse_Code.git
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd Decode_Morse_Code
+  bundle install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Koppai**
+
+- GitHub: [@KOppai](https://github.com/K0ppai)
+- LinkedIn: [Paing Soe Thu](https://www.linkedin.com/in/paingsoe-thu/)
+
+👤 **Tonny Tei**
+
+- GitHub: [@tonnytech](https://github.com/tonnytech)
+- LinkedIn: [Tonny Tei](https://www.linkedin.com/in/tonnytei/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add Database to store recent message**
+- **Add Ui**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] Kindly work on the issue link in your readme file as the link is currently broken and throwing an error 404. having a working link will make your readme file look descriptive and professional.

Issue 👇 

![image](https://github.com/K0ppai/Decode_Morse_Code/assets/78634841/e97f6c75-a628-4040-8205-84aae88a6a71)

Reply:
I think the link will work once we merge it into the development branch. @brainconnect93